manifests/a/Autodesk/Fusion360/2.0.16486/Autodesk.Fusion360.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Autodesk.Fusion360
+PackageVersion: 2.0.16486
+Installers:
+- InstallerUrl: https://dl.appstreaming.autodesk.com/production/installers/Fusion%20360%20Admin%20Install.exe
+  Architecture: x86

```

```suggestion
  Architecture: x64
```
Since `x86` validations are performed on a 32-bit OS, I'm guessing this might be a 64-bit package.